api/src/services/validation-service.ts

```diff
@@ -15,6 +19,87 @@ export class ValidationService extends DBService {
     this.validationRepository = new ValidationRepository(connection);
   }
 
+  /**
+   * Validate dataset submission
+   *
+   * @param {IDatasetSubmission} dataset
+   * @return {*}  {Promise<boolean>}
+   * @memberof ValidationService
+   */
+  async validateDatasetSubmission(dataset: IDatasetSubmission): Promise<boolean> {
```

Comment:
We should definitely come up with a good suite of unit tests for these validation functions